dm/pkg/checker/mysql_server.go

```diff
@@ -34,19 +34,14 @@ func NewMySQLVersionChecker(db *sql.DB, dbinfo *dbutil.DBConfig) RealChecker {
 }
 
 // SupportedVersion defines the MySQL/MariaDB version that DM/syncer supports
-// * 5.6.0 <= MySQL Version
-// * 10.1.2 <= Mariadb Version.
+// * 5.6.0 <= MySQL Version < 8.0.0
 var SupportedVersion = map[string]struct {
 	Min MySQLVersion
 	Max MySQLVersion
 }{
 	"mysql": {
 		MySQLVersion{5, 6, 0},
-		MaxVersion,
-	},
-	"mariadb": {
-		MySQLVersion{10, 1, 2},
-		MaxVersion,
```

Comment:
Why MariaDB is removed?

Reply:
Because MariaDB is only experimental supported.

Exist many potential problems. Same as MySQL 8.0